kevlar/assemble.py

```diff
@@ -211,7 +209,7 @@ def main(args):
         print(message, file=args.logfile)
 
     edges_dropped = prune_graph(graph)
-    cc_stream = networkx.connected_component_subgraphs(graph)
+    cc_stream = networkx.connected_component_subgraphs(graph, copy=False)
```

Comment:
Now that screed records are added to the graph as node attributes, deep copies have become problematic (see https://github.com/dib-lab/screed/issues/73). Luckily, just skipping the copy seems to work just fine.